src/applications/SimulationResults.java

```diff
@@ -67,4 +67,66 @@ public void setJobCompletionData(int jobNumber, int completionTime, int totalWai
         jobCompletions[nextJob] = jobCompletionData;
         nextJob++;
     }
+    void simulate(MachineShopSimulator machineShopSimulator) {
```

Comment:
I was a little surprised to see `simulate` move into `SimulationResults`. I guess I think of running the simulation as something the _simulator_ would do, and that would generate (as output) the _results_?